support/src/test/java/com/shopify/graphql/support/Generated.java

```diff
@@ -1111,55 +1109,67 @@ public SetIntegerInput setValue(int value) {
 
         @Nullable
         public LocalDateTime getTtl() {
+            return ttl.value;
+        }
+
+        public Input<LocalDateTime> getTtlInput() {
```

Comment:
Should we unrwap these automatically to the underlying type automatically?

Reply:
we have a separate getter, look line 1112